source/agora/flash/Node.d

```diff
@@ -1418,8 +1419,15 @@ public class FlashNode : FlashControlAPI
             return FeeUTXOs.init;
         // Always pay with the node key
         auto utxos = this.listener.getFeeUTXOs(this.conf.key_pair.address, per_byte);
+
+        // calculate a expected fee for checking the fee is correct
+        per_byte = this.listener.getEstimatedTxFee();
```

Comment:
Calling `getEstimatedTxFee` multiple times might result in different values to be returned, so it should only be called once throughout the `getFeeUTXOs`